i18n/fr.jsonp.js

```diff
@@ -137,6 +139,7 @@ document.localeJson = {
       "dialog-article-readerror-title": "Erreur lors de la lecture de l'article",
       "dialog-cancel": "Annuler",
       "dialog-ok": "OK",
+      "dialog-hide": "Ne demandez plus rien",
```

Comment:
I think "Ne demandez plus" (without "rien") is sufficient. With "rien" it means "Don't ask me anything again, ever", which is odd. Without "rien" it means "Don't ask again", which is what we want.

Reply:
Thanks again! Updated in [1c1869b](https://github.com/kiwix/kiwix-js/pull/1236/commits/1c1869b9ea6b9074457731f7c2d9d9bb14c30078).